src/GithubScrapper.php

```diff
@@ -12,6 +12,10 @@ class GithubScrapper
 {
     public const LOCATION = "location";
     public const URL = "url";
+    public const WEB = "website";
+    public const TWITTER = "twitter";
+    public const EMAIL = "email";
+    public const VERIFICATION = "verification";
```

Comment:
Maybe it will be better to move it to Backed Enum class and name it e.g. GitHubFields?
https://www.php.net/manual/en/language.enumerations.backed.php

Reply:
I think it's a overkill here. They are just learning a basics of PHP now and almost entire codebase here is going to be thrown away later.

Reply:
Got it. This is only suggestion. But in my opinion it is better to know that there are classes like Enum and if earlier you will know them then earlier you will be able to use them.